Dockerfile

```diff
@@ -41,5 +51,5 @@ RUN /home/api_checker/bin/api_checker eval ":crypto.supports()"
 
 ENTRYPOINT ["/usr/bin/dumb-init", "--"]
 
-HEALTHCHECK CMD ["/home/api_checker/bin/api_checker", "ping"]
-CMD ["/home/api_checker/bin/api_checker", "foreground"]
+HEALTHCHECK CMD ["/home/api_checker/bin/api_checker", "rpc", "1 + 1"]
```

Comment:
I gather this is some sort of hack to get the app to prove that it's still alive?

Reply:
Yep! `rpc` runs the command against the running server, so it only works if the server is online.